### ISSUE TITLE
Conda build repeats entry point when outputs is included

### ISSUE THREAD

**Issue by fpliger** [conda/conda-build#1926]
Looks like build is repeating the generation/inclusion of entry points for every output option specified, instead of just once for the top level package.

```
...
cehold_pl
number of files: 4
Fixing permissions
Fixing permissions
Detected hard-coded path in text file bin/app
Detected hard-coded path in text file bin/app-examples
Detected hard-coded path in text file bin/app_jupyter
Detected hard-coded path in text file bin/app_vanilla
/Users/fpliger/miniconda3/conda-bld/osx-64/app-assets-2.0_alpha_1-py35_0.tar.bz2
updating index in: /Users/fpliger/miniconda3/conda-bld
updating index in: /Users/fpliger/miniconda3/conda-bld/noarch
INFO conda_build.build:bundle_conda(864): Packaging app-2.0_alpha_1-py35_0
compiling .pyc files...
number of files: 470
Fixing permissions
Fixing permissions
Detected hard-coded path in text file bin/app
Detected hard-coded path in text file bin/app-examples
Detected hard-coded path in text file bin/app_jupyter
Detected hard-coded path in text file bin/app_vanilla
```

**Comment by msarahan** [comment]
It also appears that import tests for the top-level recipe are repeated for each subpackage.

**Comment by msarahan** [comment]
Fixed by #1933, closing.
